src/model_manager.py

```diff
@@ -0,0 +1,54 @@
+import json
+import requests
+
+class ModelManager:
+    def __init__(self):
+        self.useModel = False
+        self.modelURL = "http://localhost:8080/engine-rest"
+        self.process_instance_id = None
+
+    def deploy_process(self):
+        if self.useModel and self.process_instance_id is None:
+            response = requests.post(self.modelURL+"/process-definition/key/BuyVideo/start", json={})
+            response_dict = json.loads(response.text)
+            self.process_instance_id = response_dict['id']
+
+    def delete_process(self):
+        if self.useModel and self.process_instance_id is not None:
+            requests.delete(self.modelURL+"/process-instance/"+self.process_instance_id)
+            self.process_instance_id = None
+
+    
+    def complete_user_task(self, data={}, end=False):
```

Comment:
`data={}` - default value in Python should not be mutable. I'd suggest changing this to something like:
```
def complete_user_task(self, data=None, end=False):
    if data is None:
        data = {}
    # rest of the method
```

Same thing with `complete_service_task()`